docs/USAGE.md

```diff
@@ -151,6 +151,17 @@ the BindingRedirect sample can only run successfully under the instrumentation s
 binding redirect makes the application dependent on a version of `System.Diagnostics.DiagnosticSource`
 that is not available during building time.
 
+#### Additional-Deps and Runtime Package Store
+
+To resolve assembly version conflicts in .NET Core, set the environment variables
+[DOTNET_ADDITIONAL_DEPS](https://github.com/dotnet/runtime/blob/main/docs/design/features/additional-deps.md)
+and [DOTNET_SHARED_STORE](https://docs.microsoft.com/en-us/dotnet/core/deploying/runtime-store) to below value.
+
+| Environment variable | Value |
+|-|-|
+| `DOTNET_ADDITIONAL_DEPS` | tracer-home\AdditionalDeps |
+| `DOTNET_SHARED_STORE` | tracer-home\store |
```

Comment:
1. `OTEL_DOTNET_AUTO_HOME` describes `tracer-home` as `Installation location.`
2. I prefer using `/` instead of `\` when somethin is not tightly-coupled to Windows

```suggestion
| `DOTNET_ADDITIONAL_DEPS` | `%InstallationLocation%/AdditionalDeps` |
| `DOTNET_SHARED_STORE` | `%InstallationLocation%/store` |

Where `%InstallationLocation%` stands for the OpenTelemetry .NET Auto-Instrumentation installation location.
```